### DISCUSSION TITLE
paramountplus.com: breakage

### DISCUSSION THREAD

**Discussion by terrencetingleberry** [uBlockOrigin/uAssets#23560] [General]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.paramountplus.com/home/`

### Category

breakage

### Description

On Paramount+, there are many similar "WATCH NOW", "RESUME", and other buttons of that sort, but with my uBO config, these buttons don't work anymore. After extensive testing, I finally figured out if I specifically set an allowance for **ALL** 3rd-party scripts on **paramountplus.com** and disable **'AdGuard/uBO – Cookie Notices'**, these buttons function properly.

I'm not sure exactly which 3rd-party scripts are responsible for the function of these buttons, nor do I know what in **'AdGuard/uBO – Cookie Notices'** is responsible. Are there any specific exclusions I can set for Paramount+ which would allow the buttons to function properly, without me needing to let unwanted cookie notices or 3rd-party scripts through? Also, I'm aware I did technically enable additional filter lists, but I don't know how to solve this issue other than to seek help here; Also, even if I were to disable all additional filter lists (or just **'AdGuard/uBO – Cookie Notices'**), the 3rd-party scripts would still be an issue.

### Other extensions used

- Bitwarden - Free Password Manager
- Dark Reader
- Enhancer for YouTube™
- Extension source viewer (by Rob W)
- Firefox Multi-Account Containers
- nightTab
- Skip Redirect (by Sebastian Blask)

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screenshot from 2024-04-28 18-55-54](https://github.com/uBlockOrigin/uAssets/assets/142938968/37c8384c-4b2e-488c-b670-191bdb72e5f6)
![image](https://github.com/uBlockOrigin/uAssets/assets/142938968/7a523c5f-fa11-4430-96a8-21e18b4ab5d6)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.57.2
Firefox: 125
filterset (summary):
 network: 234360
 cosmetic: 243569
 scriptlet: 50444
 html: 2042
listset (total-discarded, last-updated):
 added:
  https://raw.githubusercontent.com/DandelionSprout/adfilt/master/LegitimateURLShortener.txt: 2623-0, 1h.37m
  adguard-generic: 81128-3930, now
  adguard-spyware-url: 1503-235, now
  adguard-spyware: 80368-1339, now
  block-lan: 66-0, now
  curben-phishing: 14025-2, now
  adguard-cookies: 29370-45, now
  ublock-cookies-adguard: 1084-13, now
  fanboy-cookiemonster: 50975-3872, now
  ublock-cookies-easylist: 1084-1084, now
  adguard-social: 22875-55, now
  fanboy-social: 15835-724, now
  [10 lists not shown]: [too many]
 default:
  user-filters: 1-0, never
  ublock-filters: 37818-109, 1h.37m Δ
  ublock-badware: 8429-0, 1h.37m Δ
  ublock-privacy: 833-52, 1h.37m Δ
  ublock-unbreak: 2321-2, 1h.37m Δ
  ublock-quick-fixes: 191-25, 1h.37m Δ
  easylist: 86770-1627, 1h.37m Δ
  easyprivacy: 50823-25487, 1h.37m Δ
  urlhaus-1: 13010-0, now
  plowe-0: 3735-1243, now
filterset (user): [array of 1 redacted]
hostRuleset:
 added: [array of 61 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 416 ms (selfie)
 maxAssetCacheWait: 249 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 23
 network:
  appboycdn.com: 2
  braze.com: 1
  cookielaw.org: 1
  jsdelivr.net: 4
  newrelic.com: 4
  paramount.com: 1
  storage.googleapis.com: 10
 extended:
  ##+js(set-constant, hasAdBlocker, false)
  ##+js(xml-prune, Period[id*="-roll-"][id*="-ad-"], , pubads.g.do…
```
</details>

**Comment by u-RraaLL** [comment]
I've converted this to a discussion, because according to your description this is not an issue with default config.

Use [the logger](https://github.com/gorhill/uBlock/wiki/The-logger) to determine what you need to noop in 3p scripts.

And if you're able to determine which filter breaks AdGuard/uBO – Cookie Notices (with the same tool), we can add an exception to the list.

We can't figure this out for you since the site requires a subscription.
